optimize/convex/lp/simplex.go

```diff
@@ -100,6 +100,19 @@ func simplex(initialBasic []int, c []float64, A mat.Matrix, b []float64, tol flo
 	}
 	m, n := A.Dims()
 
+	if m == n {
+		// Problem is exactly constrained, perform a linear solve.
+		bVec := mat.NewVecDense(len(b), b)
+		x := make([]float64, m)
```

Comment:
`m == n` here but still it is strange to use `m` and `n` on the next line.

Reply:
Done.